scripts/monorepo/compass-deps-list.js

```diff
@@ -0,0 +1,400 @@
+  },
+  {
+    name: 'electron-squirrel-startup',
+    description:
+      'Default Squirrel.Windows event handler for your Electron apps.',
+    repository: 'https://github.com/mongodb-js/electron-squirrel-startup.git'
+  },
+  {
+    name: 'hadron-app',
+    description: 'Hadron Application Singleton',
+    repository: 'https://github.com/mongodb-js/hadron-app.git'
+  },
+  {
+    name: 'hadron-app-registry',
+    description: 'Hadron App Registry',
+    repository: 'https://github.com/mongodb-js/hadron-app-registry.git'
+  },
+  {
+    name: 'hadron-auto-update-manager',
+    description: 'Atoms AutoUpdateManager class as a standalone module.',
+    repository: 'https://github.com/mongodb-js/hadron-auto-update-manager.git'
+  },
+  {
+    name: 'hadron-build',
+    description: 'Tooling for Hadron apps.',
+    repository: 'https://github.com/mongodb-js/hadron-build.git'
+  },
+  {
+    name: 'hadron-compile-cache',
+    description: 'Hadron Compile Cache',
+    repository: 'https://github.com/mongodb-js/hadron-compile-cache.git'
+  },
+  {
+    name: 'hadron-document',
+    description: 'Hadron Document',
+    repository: 'https://github.com/mongodb-js/hadron-document.git'
+  },
+  {
+    name: 'hadron-ipc',
+    description: 'Simplified IPC for electron apps.',
+    repository: 'https://github.com/mongodb-js/hadron-ipc.git'
+  },
+  {
+    name: 'hadron-module-cache',
+    description: 'Hadron Module Cache',
+    repository: 'https://github.com/mongodb-js/module-cache.git'
+  },
+  {
+    name: 'hadron-plugin-manager',
+    description: 'Hadron Plugin Manager',
+    repository: 'https://github.com/mongodb-js/hadron-plugin-manager.git'
+  },
+  {
+    name: 'hadron-react-bson',
+    description: 'Hadron React BSON Components',
+    repository: 'https://github.com/mongodb-js/hadron-react.git'
+  },
+  {
+    name: 'hadron-react-buttons',
+    description: 'Hadron React Button Components',
+    repository: 'https://github.com/mongodb-js/hadron-react.git'
+  },
+  {
+    name: 'hadron-react-components',
+    description: 'Hadron React Components',
+    repository: 'https://github.com/mongodb-js/hadron-react.git'
+  },
+  {
+    name: 'hadron-react-utils',
+    description: 'Hadron React Utils',
+    repository: 'https://github.com/mongodb-js/hadron-react.git'
+  },
+  {
+    name: 'hadron-spectron',
+    description: 'Hadron Spectron Test Support',
+    repository: 'https://github.com/mongodb-js/hadron-spectron.git'
+  },
+  {
+    name: 'hadron-style-manager',
+    description: 'Hadron Style Manager',
+    repository: 'https://github.com/mongodb-js/hadron-style-manager.git'
+  },
+  {
+    name: 'hadron-type-checker',
+    description: 'Hadron Type Checker',
+    repository: 'https://github.com/mongodb-js/hadron-type-checker.git'
+  },
+  {
+    name: 'mongodb-ace-autocompleter',
+    description: 'Ace Editor Autocompleter for MongoDB Queries & Agg Pipelines',
+    repository: 'https://github.com/mongodb-js/ace-autocompleter.git'
+  },
```

Comment:
Also shared with mongosh, fwiw

Reply:
Yep, let's remove it also!

Reply:
Gone in fd579d4 although looking at how `mongosh` is using it I wonder if all this "grammar" that `mongosh` imports just belongs somewhere else separately from the ace part of the package (but that's probably a discussion for another time 😄)

Reply:
@gribnoysup that's a great point, it would be good to split the grammar from the actual ACE part and keep the more UI part in compass. Doesn't even seem such an heavy task, how do we feel creating a ticket for that to be done separately?

Reply:
Sure, will make a ticket for this!